src/session/session_api.c

```diff
@@ -833,16 +833,42 @@ __session_alter_readonly(WT_SESSION *wt_session, const char *uri, const char *co
 int
 __wt_session_count(WT_SESSION *wt_session, const char *uri, int64_t *count)
```

Comment:
This api only supports row count not byte count?

Reply:
Yes, that's correct. I need to clarify with Vamsi whether it's better to add this as a different API or into the same one.